shoop/admin/modules/products/views/edit_media.py

```diff
@@ -21,6 +23,7 @@
 from shoop.utils.multilanguage_model_form import MultiLanguageModelForm
 
 
+# XXX: Will be deprecated when product edit view has other media management
```

Comment:
Are these remarks necessary? We'll delete things when the issue for that rolls around anyway.